knora-ontologies/standoff-onto.ttl

```diff
@@ -97,7 +99,19 @@
                               owl:maxCardinality "1"^^xsd:nonNegativeInteger
                            ] ;
 
-                        rdfs:comment "Represents a hyperlink in a text"@en .
+            rdfs:comment "Represents a hyperlink in a text"@en .
+
+###  http://www.knora.org/ontology/standoff#StandoffBlockquoteTag
+
+:StandoffBlockquoteTag rdf:type owl:Class ;
+
+            rdfs:subClassOf :StandoffHyperlinkTag ,
```

Comment:
how do we tell... it is a link in the end, but there is a difference somehow where the `StandoffUriTag` is a primarily a link with a label, as the `StandoffBlockquoteTag` is more a part of a text, with its source.

Removing the attribute definition